packages/pn-personagiuridica-webapp/src/components/Notifications/GroupSelector.tsx

```diff
@@ -33,7 +35,11 @@ const GroupSelector: React.FC<Props> = ({ currentGroup, onGroupSelection }) => {
   }, []);
 
   return (
-    <Box data-testid="groupSelector" display="inline-block" sx={{ verticalAlign: 'middle', mx: 2 }}>
+    <Box
+      data-testid="groupSelector"
+      display="inline-block"
+      sx={{ verticalAlign: 'middle', mx: isMobile ? 0 : 2 }}
```

Comment:
change this property with `sx={{ verticalAlign: 'middle', mx: { xs: 0, lg: 2 }, mt: { xs: 1, lg: 0 } }}`

Reply:
done